src/module.c

```diff
@@ -12717,6 +12717,18 @@ int RM_RdbLoad(RedisModuleRdbStream *stream, int flags) {
         return REDISMODULE_ERR;
     }
 
+    /* Not allowed on replicas. */
+    if (server.masterhost != NULL) {
+        errno = EPERM;
```

Comment:
EPERM could also be a file system error. maybe we can find another error that more suitable?
maybe EINVAL would be appropriate (unlikely to happen in normal use, unless there's a bug in redis)?

Reply:
looks like fopen() returns EINVAL if filename consists invalid characters. Maybe ENOTSUP is better?